ISSUE_TEMPLATE.md

```diff
@@ -1,19 +1,58 @@
+<!--- Keep everything below and click 'Submit new issue'  --->
+
 ### What problem are we trying to solve?
 
 Answer here.
 
+
 ### Who will benefit (directly and indirectly) from this project?
 
 Answer here.
 
-### Links to any research/data available/articles
+
+### Where can we find any research/data available/articles?
 
 Links here.
 
-### What are the next steps (validation, research, coding, design)? 
+
+### What help is needed at this time?
 
 Answer here.
 
-### What help is needed at this time?
+
+### What are the next steps (validation, research, coding, design)? 
 
 Answer here.
+
+---- 
+<!--- You can skip the following for now, but come back to it later!  --->
+
+### Project management
+
+#### Checklist for NEW ideas :baby:
+Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).
+
+- [x] Create this idea issue
+- [x] Flesh out the who, where, and what questions above
+- [ ] Start the conversation about this new idea on [Slack](https://openaustin-slackin.herokuapp.com/)!
+- [ ] Respond with at least one update on this issue within the next month
+
+#### Checklist for ACTIVE projects :fire:
+Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:
+
+- [ ] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
+- [ ] Create a [GitHub repository](https://github.com/new) and Slack channel for work.
+- [ ] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
+- [ ] Turn on your [Issue Tracker](https://mozilla.github.io/global-sprint/project-lead/templates/#4-create-issues) and create issues to describe each task that you need help with and how a contributor can get started on that task.
```

Comment:
You mean all four steps? The Open Leadership 101 is a way to ensure that folks who want to take an idea to a project know some fundamentals of open project management and know of this source as a guide for reference later. The README is the closest thing to a front page for non-GIthub savvy users to using a Github repo, so it's extremely important, as is the issue tracker for providing an open area for collaboration that is also more permanent and trackable than Slack. Trello could also go here, but I don't want to make folks think they need so many different tools to get a project started.